src/Spectre.AngularClient/src/app/preparations/preparation/preparation.component.html

```diff
@@ -5,9 +5,18 @@
 <md-slider class="slider" [(ngModel)]="currentChannelId" showTicks="true" min="0" max={{mzLenth}} step="1" (input)="onInputChannelId($event)"
            (change)="onChangedChannelId($event)" vertical="false" thumb-label="true" tick-interval="true"></md-slider>
 
-<app-heatmap
-  [heatmapData]=heatmapData>
-</app-heatmap>
+<div>
+  <div class="inlineItemsContainer">
+    <md-slider class="sliderY" [(ngModel)]="yCoordinate" showTicks="true" min="0" max={{yHeatmapSize}} step="1" vertical="true" thumb-label="true" tick-interval="true"
+           (change)="onChangedYCoordinate($event)"></md-slider>
+
+    <app-heatmap
+      [heatmapData]=heatmapData>
+    </app-heatmap>
+  </div>
+  <md-slider class="sliderX" [(ngModel)]="xCoordinate" showTicks="true" min="0" max={{xHeatmapSize}} step="1" vertical="false" thumb-label="true" tick-interval="true"
```

Comment:
I'd go with this over the heatmap, and `channelId` slider below the heatmap, so it is just over the spectrum.